pkg/stconfig/keygen.go

```diff
@@ -0,0 +1,19 @@
+package stconfig
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/u-root/u-root/pkg/crypto"
+)
+
+// GenKeys generates ED25519 keypair and stores it on the harddrive
+func GenKeys(genkeysPrivateKeyFile, genkeysPublicKeyFile, genkeysPassphrase string) error {
+	if _, err := os.Stat(genkeysPrivateKeyFile); os.IsNotExist(err) {
```

Comment:
I don't entirely understand this. The file must already exist before GenerateED25519Key creates it?